raiden/transfer/log.py

```diff
@@ -83,6 +84,12 @@ def __init__(self, database_path):
             'FOREIGN KEY(statechange_id) REFERENCES state_changes(id)'
             ')'
         )
+        cursor.execute(
+            'CREATE TABLE IF NOT EXISTS state_events ('
+            'identifier integer primary key, source_statechange_id integer NOT NULL, data binary, '
+            'FOREIGN KEY(source_statechange_id) REFERENCES state_changes(id)'
+            ')'
+        )
         self.conn.commit()
 
     def write_transaction(self, data):
```

Comment:
Hmm I got pyflakes and it does not catch the error. Pylinter I still have not integrated with my editor will do at some point.

Will rename and fix the signature mismatch.

As for the batch transactions this is not a requirement right now and may happen in the future. We have both a comment there and an issue: https://github.com/raiden-network/raiden/issues/587